addons/l10n_nl/data/account_account_tag.xml

```diff
@@ -1,54 +1,138 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<odoo>
     <data noupdate="1">
 
         <!-- Account Tags -->
         <record id="account_tag_1" model="account.account.tag">
-            <field name="name">Lonen &amp; Salarissen</field>
+            <field name="name">Immateriële vaste activa</field>
             <field name="applicability">accounts</field>
         </record>
         <record id="account_tag_2" model="account.account.tag">
-            <field name="name">Huisvestingkosten</field>
+            <field name="name">Materiële vaste activa</field>
             <field name="applicability">accounts</field>
         </record>
         <record id="account_tag_3" model="account.account.tag">
-            <field name="name">Afschrijvingen machines, installaties en machines</field>
+            <field name="name">Financiële vaste activa</field>
             <field name="applicability">accounts</field>
         </record>
         <record id="account_tag_4" model="account.account.tag">
-            <field name="name">Gas, water en electriciteit</field>
+            <field name="name">Voorraden</field>
             <field name="applicability">accounts</field>
         </record>
         <record id="account_tag_5" model="account.account.tag">
-            <field name="name">Kantoor</field>
+            <field name="name">Vorderingen</field>
             <field name="applicability">accounts</field>
         </record>
         <record id="account_tag_6" model="account.account.tag">
-            <field name="name">Auto</field>
+            <field name="name">Effecten</field>
             <field name="applicability">accounts</field>
         </record>
         <record id="account_tag_7" model="account.account.tag">
-            <field name="name">Verkoopkosten</field>
```

Comment:
You rename account_tag_7 from Verkoopkosten to Liquide middelen and then create a new tag account_tag_25 Verkoopkosten. This seems like a migration nightmare for poor @odony or whoever is doing that these days. How about keeping account_tag_7 as it is and add Liquide middelen as a new tag? There are other examples of this as well.